azure-pipeline-weekly.yml

```diff
@@ -27,18 +27,25 @@ stages:
         inputs:
           useGlobalJson: true
 
+      - task: UseDotNet@2
+        displayName: Install .NET 3.1 runtime
+        inputs:
+          packageType: runtime
+          version: 3.1.x
+
       - script: dotnet build
         workingDirectory: src/Microsoft.DncEng.SecretManager
 
       - task: AzureCLI@2
         inputs:
-          azureSubscription: NetHelix
+          azureSubscription: DotNet Eng Services Secret Manager
           scriptType: ps
           scriptLocation: inlineScript
           inlineScript: |
             Get-ChildItem .vault-config/*.yaml |% { dotnet run -p src/Microsoft.DncEng.SecretManager -- synchronize $_}
 
   - stage: Build
+    dependsOn:
```

Comment:
Wasn't this empty dependency forgot to be deleted?

Reply:
its intentionally empty. Not specifying dependsOn makes it default to "the previous stage in the yaml". I want there to not be a dependency here.